staging/src/k8s.io/apiextensions-apiserver/pkg/controller/openapi/builder/builder.go

```diff
@@ -424,40 +456,46 @@ func addEmbeddedProperties(s *spec.Schema, v2 bool) {
 
 // getDefinition gets definition for given Kubernetes type. This function is extracted from
 // kube-openapi builder logic
-func getDefinition(name string) spec.Schema {
-	buildDefinitions.Do(buildDefinitionsFunc)
+func getDefinition(name string, v2 bool) spec.Schema {
+	buildDefinitions.Do(generateBuildDefinitionsFunc(v2))
 	return definitions[name].Schema
 }
 
 func withDescription(s spec.Schema, desc string) spec.Schema {
 	return *s.WithDescription(desc)
 }
 
-func buildDefinitionsFunc() {
-	namer = openapi.NewDefinitionNamer(runtime.NewScheme())
-	definitions = utilopenapi.GetOpenAPIDefinitionsWithoutDisabledFeatures(generatedopenapi.GetOpenAPIDefinitions)(func(name string) spec.Ref {
-		defName, _ := namer.GetDefinitionName(name)
-		return spec.MustCreateRef(definitionPrefix + common.EscapeJsonPointer(defName))
-	})
+func generateBuildDefinitionsFunc(v2 bool) func() {
+	return func() {
+		namer = openapi.NewDefinitionNamer(runtime.NewScheme())
+		definitions = utilopenapi.GetOpenAPIDefinitionsWithoutDisabledFeatures(generatedopenapi.GetOpenAPIDefinitions)(func(name string) spec.Ref {
```

Comment:
```suggestion
		definitions = utilopenapi.GetOpenAPIDefinitionsWithoutDisabledFeatures(generatedopenapi.GetOpenAPIDefinitions)(func(name string) spec.Ref {
```

Reply:
To fix the gofmt 😄

Reply:
This is an outdated comment right? The verify failure was before the last force push

Reply:
Could you point out what got changed?

Reply:
Oh I see. already outdated.

Reply:
Oh, outdated. Sorry.